src/main/scala/treadle/stage/phases/GetFirrtlAst.scala

```diff
@@ -29,8 +29,11 @@ object GetFirrtlAst extends Phase {
     def handleFirrtlSource(): Option[AnnotationSeq] = {
       annotationSeq.collectFirst { case FirrtlSourceAnnotation(firrtlText) => firrtlText } match {
         case Some(text) =>
+
+          //Run the additional coverage firrtl passes if needed
```

Comment:
I think this comments got left behind after last round. I think GetFirrtlAst doesn't need any changes now that the pass is schedule using the standard stage/phase mode.

Reply:
woops yeah I didn't see that there, my bad ^^"